.github/workflows/sync_from_airtable.yml

```diff
@@ -0,0 +1,325 @@
+                            labels: addLabels
+                        })
+                        core.info(`Labels added: ${addLabels}`)
+                    } catch (error) {
+                        if (error.toString().includes('Label does not exist')) {
+                            await Promise.all(addLabels.map(async (label) => {
+                                try {
+                                    await github.rest.issues.getLabel({
+                                        owner: OWNER,
+                                        repo: REPO,
+                                        name: label
+                                    });
+                                    await github.rest.issues.addLabels({
+                                        owner: OWNER,
+                                        repo: REPO,
+                                        issue_number: NUMBER,
+                                        labels: label
+                                    });
+                                    core.info(`Label added: ${label})`)
+                                } catch (error) {
+                                    try {
+                                        await github.rest.issues.createLabel({
+                                            owner: OWNER,
+                                            repo: REPO,
+                                            name: label
+                                        });
+                                        core.info(`Label created: ${label}`)
+                                        await github.rest.issues.addLabels({
+                                            owner: OWNER,
+                                            repo: REPO,
+                                            issue_number: NUMBER,
+                                            labels: label
+                                        });
+                                        core.info(`Label added: ${label}`)
+                                    } catch (error) {
+                                        console.error(`Label not added: ${error}`)
+                                    }
+                                }
+                            }))
+                        }
+                    }
+                }
+                if (removeLabels && removeLabels.length > 0) {
+                    try {
+                        await Promise.all(removeLabels.map(async (name) => {
+                            await github.rest.issues.removeLabel({
+                                owner: OWNER,
+                                repo: REPO,
+                                issue_number: NUMBER,
+                                name
+                            });
+                            core.info(`Remove Label: ${name}`)
+                        }));
+                    } catch (err) {
+                        if (!error.includes('Label does not exist')) { core.error(`Label not removed: ${err}`) }   
```

Comment:
For consistency, can you move the if statement contents to its own line?

Reply:
fixed